packages/webdriverio/src/commands/element/click.ts

```diff
@@ -88,7 +88,7 @@ export default async function click(
     options?: ClickOptions
 ) {
     if (typeof options === 'undefined') {
-        return this.elementClick(this.elementId)
+        options = {}
```

Comment:
This isn't the way to assign a default object, instead it should be:
```js
export default async function click(
    this: WebdriverIO.Element,
    options?: ClickOptions = {}
) {
```
After which this part can be removed.

However, doing this completely changes the behavior of the click command and I know that @christian-bromann wanted to have the elementClick in here specifically when no options are passed. So Christian, do we still want this?